src/@ionic-native/plugins/ble/index.ts

```diff
@@ -417,6 +417,33 @@ export class BLE extends IonicNativePlugin {
   @Cordova()
   isEnabled(): Promise<void> { return; }
 
+  /**
+   * Register to be notified when Bluetooth state changes on the device.
+   *
+   * @usage
```

Comment:
Please remove the usage block from here. This will not be displayed in the actual docs. You should document the method in the JSDoc of the class.

Reply:
Never mind, the whole file is not formatted properly. I will fix it later.